dotCMS/src/main/java/com/dotmarketing/servlets/BinaryExporterServlet.java

```diff
@@ -523,15 +507,14 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) throws Servl
 							out.println();
 							out.println("--" + SpeedyAssetServletUtil.MULTIPART_BOUNDARY + "--");
 						}
-						responseSent = true;
 					}
 				} catch (Exception e) {
 					Logger.warn(this, e + " Error for = " + req.getRequestURI() + (req.getQueryString() != null?"?"+req.getQueryString():"") );
 					Logger.debug(this, "Error serving asset = " + req.getRequestURI() + (req.getQueryString() != null?"?"+req.getQueryString():""), e);
 
-				} 
+				}
 			}else{
-				is = new FileInputStream(data.getDataFile());
+				is = java.nio.file.Files.newInputStream(data.getDataFile().toPath());

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a "try/catch" block for "newInputStream". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1989)